src/MetadataUtility/Models/Microphone.cs

```diff
@@ -39,5 +39,10 @@ public record Microphone
         /// Gets the name of the microphone (style depends on vendor).
         /// </summary>
         public string ChannelName { get; init; }
+
+        /// <summary>
+        /// Gets the sensitivity of the microphone.
+        /// </summary>
+        public string Sensitivity { get; init; }
```

Comment:
we should make a note of the correct SI unit for this and make sure we're reporting the units correctly. Also document the unit which this value is in.